sql/src/SqlSchemaMigration.sk

```diff
@@ -16,16 +16,13 @@ private fun resetReactiveViews(
   options: Options,
 ): void {
   viewsDir = context.unsafeGetEagerDir(getViewsDir(context).dirName);
-  viewsToReset = viewsDir
-    .unsafeGetFileIter()
-    .flatMap((x) ->
-      x.i1.filter((file) ->
-        SelectFile::type(file).cselect.from.any((from) ->
-          alteredDirs.contains(from.i0.name)
-        )
+  viewsToReset = viewsDir.unsafeGetFileIter().flatMap((x) ->
+    x.i1.filter((file) ->
+      SelectFile::type(file).cselect.from.any((from) ->
+        alteredDirs.contains(from.i0.name)
       )
     )
-    .collect(Array);
+  );
```

Comment:
The failure is due to this change, I'm investigating

Reply:
Do you have a stack trace?